<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>-50</y>
      <width>1391</width>
      <height>871</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>FreeMono</family>
         <pointsize>22</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shape</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>choose the one you want</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <item>
        <property name="text">
         <string>Clear scene</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rhombus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hexagon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>150</y>
      <width>263</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>FreeMono</family>
         <pointsize>22</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Transformation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Rotate:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Up/down:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Right/left:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>FreeMono</family>
         <pointsize>22</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Star</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>choose number of angles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>140</y>
      <width>20</width>
      <height>501</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>update perimeter</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>470</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>FreeMono</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Perimeter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>tap button to update</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>FreeMono</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Area:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>tap button to update</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>update area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>420</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FreeMono</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
